src/main/java/io/fabric8/vertx/maven/plugin/utils/PackageHelper.java

```diff
@@ -134,6 +146,8 @@ protected void generateManifest() {
             attributes.put(MAIN_VERTICLE, mainVerticle);
         }
 
+        ManifestUtils.addExtraManifestInfo(mavenProject, attributes);
```

Comment:
That made me thinks to the case we use an existing manifest as input. Not something to handle in this PR, but something we should track.